### ISSUE TITLE
Add `Input.get_pen_pressure()` method

### ISSUE THREAD

**Issue by mieldepoche** [godotengine/godot-proposals#7620]
### Describe the project you are working on

prototype involving a drawing tablet

### Describe the problem or limitation you are having in your project

Right now the only way to get pen pressure (*to my knowledge*) is when receiving an `InputMouseMotion` event, which contains a `pressure` member.
This is limited since pressure can change without mouse/pen movement, and it makes it ugly to have to store the value you receive when you need to use it outside of `_input*` functions.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Add a `get_pen_pressure()` (or `get_mouse_pressure()`) method to `Input`.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

```gdscript
func _process(_delta: float) -> void:
    print(Input.get_pen_pressure())
```

### If this enhancement will not be used often, can it be worked around with a few lines of script?

it partially can be hacked around, but you can't track pressure change when the mouse/pen doesn't move: clicking a button on the mouse changes the pressure, but if you don't move the mouse you never hear about it.

### Is there a reason why this should be core and not an add-on in the asset library?

it can't

**Comment by Mickeon** [comment]
Good point. It _feels_ like the **Input** singleton should have access to this information. Calling it `Mouse pressure` would be quite silly, though.
While I do agree, is there a more concrete example? It's the first time I've heard of someone needing this.

**Comment by mieldepoche** [comment]
I though about it a bit and after testing, it turns out none of the limitations I thought were there exist :sweat_smile::
- you *can* track pressure change on mouse clicks without mouse movement by just checking that `pressed` state of the mouse button event (I didn't think of that for some reason initially...),
- with a tablet, pen pressure change without pen movement actually generate new `InputEvenMouseMotion` with the new pressure, but with no movement data (`ev.relative == Vector2.ZERO`) which is not intuitive but it works I guess.

It would still make for an easy way to fetch pen pressure every frame (I needed that for a shader parameter) instead of having to store it in a variable and check mousebutton events just in case, but now the more I think about it now, the less I'm convinced.

Also as a side note, it this was to be added, there should also be a getter for other pen properties (tilt value and eraser mode).

**Comment by Nintorch** [comment]
Hello, is this feature still needed? I can try to make a method that returns the value from the events by storing it internally, if that solution is good enough

**Comment by mieldepoche** [comment]
As I realized in my comment above, there are actually no limitations with the current way things work.
The only benefit `Input.get_pen_pressure()` would give is, a simpler way to get pen pressure every frame. And it would "squash" the input events to only one pressure value per frame, which you probably don't want if you're making a drawing app.

> is this still needed?

I would say no.
